docs/using/python.md

```diff
@@ -10,164 +10,161 @@ has_children: false
 # Calling the lakeFS API from Python
 {: .no_toc }
 
-The [lakeFS API](../reference/api.md){: target="_blank" } is OpenAPI 2.0 compliant, allowing the dynamic generation of clients from multiple languages.
+The [lakeFS API](../reference/api.md){: target="_blank" } is OpenAPI 3.0 compliant, allowing the generation of clients from multiple languages.
 
-For Python, this example uses [Bravado](https://github.com/Yelp/bravado){: target="_blank" }
-which generates a dynamic client at runtime, from an OpenAPI definition served by a lakeFS server.
+For Python, this example uses [lakeFS's python package](https://pypi.org/project/lakefs-client/){: target="_blank" }.
+The lakefs-client pacakge was created by [OpenAPI Generator](https://openapi-generator.tech){: target="_blank" } using our OpenAPI definition served by a lakeFS server.
 
 ## Table of contents
 {: .no_toc .text-delta }
 
 1. TOC
 {:toc}
 
-## Installing Python Dependencies
+## Install lakeFS Python Client API
+
+Install the Python client using pip:
 
-A complete installation guide is available in the [bravado GitHub repository](https://github.com/Yelp/bravado){: target="_blank" }.
-For our example, we'll simply install it with pip:
 
 ```shell
-pip install bravado==10.6.2
+pip install 'lakefs_client==<lakeFS version>'
 ```
 
-At the time of writing this guide, the current stable bravado release is `10.6.2`.
-
-
-## Locating the OpenAPI definition for our installation
+The package is available from version > 0.33.1.
 
-Assuming we have a lakeFS server deployed at `https://lakefs.example.com`, the OpenAPI definition URL will be `https://lakefs.example.com/swagger.json`.
-For local development, we can use `http://localhost:8000/swagger.json`.
 
-## Generating a Python client with Bravado
-
-Once we have Bravado installed, and a URL of an OpenAPI definition we can generate a client:
+## Working with the Client API
 
+How to instantiate a client:
 
 ```python
-from bravado.requests_client import RequestsClient
-from bravado.client import SwaggerClient
+import lakefs_client
+from lakefs_client import models
+from lakefs_client.client import LakeFSClient
 
-http_client = RequestsClient()
-http_client.set_basic_auth('localhost:8000', 'AKIAIOSFODNN7EXAMPLE', 'wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY')
-client = SwaggerClient.from_url('http://localhost:8000/swagger.json', http_client=http_client)
+configuration = lakefs_client.Configuration()
+configuration.username = 'AKIAIOSFODNN7EXAMPLE'
+configuration.password = 'wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY'
+configuration.host = 'http://localhost:8000/api/v1'
```

Comment:
Add a comment describing that these are the values for your lakeFS installation.
Might be trivial to us, but users often mix up these values.